.github/workflows/core.test+build.yaml

```diff
@@ -25,7 +25,8 @@ jobs:
         run: yarn test
         working-directory: ./core
         env:
-          ORAKL_DIR: './'
+          NODE_ENV: "development"
```

Comment:
I think it would be safer to not change `NODE_ENV` to `development`. There are good reasons why the `NODE_ENV` is set to `test` when running tests.

 It would be better to use Github Actions specific environment variables, because that is the goal anyway; knowing when we are using pipeline and when not.

https://docs.github.com/en/github-ae@latest/actions/learn-github-actions/variables#default-environment-variables

You could use `GITHUB_ACTIONS` instead.

> Always set to `true` when GitHub Actions is running the workflow. You can use this variable to differentiate when tests are being run locally or by GitHub Actions.